docs/cookbook/entities/map.rst.inc

```diff
@@ -1,3 +1,4 @@
 * :doc:`/cookbook/entities/first-resource`
 * :doc:`/cookbook/entities/manage-your-entity`
 * :doc:`/cookbook/entities/configure-your-routes`
+* :doc:`/cookbook/entities/configure-admin-menu`
```

Comment:
you also needs to add this entry on menu (cf Travis) to access to this cookbook.

Reply:
Not sure I get you here. Can you point where to add ? Not seeing anything on travis.yml .

Reply:
Oh, I found it. Adding.

Reply:
You need to add this page on this [tree](https://github.com/Monofony/SymfonyStarter/blob/master/docs/cookbook/index.rst#entities).

```
.. toctree::
    :hidden:

    entities/first-resource
    entities/manage-your-entity
    entities/configure-your-routes
    entities/configure-admin-menu
```

Reply:
Thank you. Already added :) . Found it a few mins before via search.